tests/e2e/default/sms/rapidpro.wdio-spec.js

```diff
@@ -302,7 +296,6 @@ describe('RapidPro SMS Gateway', () => {
 
     afterEach(() => utils.revertDb([], true));
```

Comment:
If it is updating settings in the beforeEach hook, can you please check if adding a revertSetting in the afterEach hook or maybe the afterHook is necessary? 
The updateSettings also have an option to revertSettings before updating.

Reply:
I added the `revertSettings` in the `afterEach` from the first `describe`, in that way it will be valid for all the `it`, and also removed this `revertDb` call because it is being duplicated with the `afterEach` from the first describe. Thank you Lore!